src/sync/pull-requests.test.ts

```diff
@@ -471,6 +473,59 @@ describe("sync/pull-request", () => {
 			removeInterceptor(interceptor);
 		});
 
+		it("should only use pull requests that are later than fromDate is supplied -- when ff is on", async () => {
+
+			when(booleanFlag).calledWith(
+				BooleanFlags.USE_BACKFILL_ALGORITHM_INCREMENTAL,
+				jiraHost
+			).mockResolvedValue(true);
+
+			githubUserTokenNock(DatabaseStateCreator.GITHUB_INSTALLATION_ID);
+
+			const mockPullRequestList = () => {
+				githubNock
+					.get("/repos/integrations/test-repo-name/pulls?per_page=20&page=1&state=all&sort=created&direction=desc")
+					.reply(200, [
+						{ ...pullRequest, title: "PR3", created_at: "2023-01-03T00:00:00Z" },
+						{ ...pullRequest, title: "PR2", created_at: "2023-01-02T00:00:00Z" },
+						{ ...pullRequest, title: "PR1", created_at: "2023-01-01T00:00:00Z" }
+					]);
+			};
+
+			mockPullRequestList();
+
+			const gitHubClient = await createInstallationClient(DatabaseStateCreator.GITHUB_INSTALLATION_ID, jiraHost, getLogger("test"), undefined);
+			expect(await doGetPullRequestTask(getLogger("test"),
+				gitHubClient,
+				jiraHost,
+				{
+					id: repoSyncState.repoId,
+					name: repoSyncState.repoName,
+					full_name: repoSyncState.repoFullName,
+					owner: { login: repoSyncState.repoOwner },
+					html_url: repoSyncState.repoUrl,
+					updated_at: repoSyncState.repoUpdatedAt?.toISOString()
+				},
+				undefined,
+				20,
+				{
+					jiraHost,
+					installationId: DatabaseStateCreator.GITHUB_INSTALLATION_ID,
+					commitsFromDate: "2023-01-01T01:02:03Z"
```

Comment:
For now, since the "restart/continue backkflll" from UI only allow user to pick one date, we will just reuse that date. Later if UI can pick multiple dates, we can use the pull request from date.